README.md

```diff
@@ -24,7 +24,7 @@ mvn clean install
 Execute Command under a project that you want to advise for jakarta 10
 
 ```
-mvn fish.payara.advisor:advisor-maven-plugin:1.0-SNAPSHOT:advisor-tool
+mvn fish.payara.advisor:advisor-maven-plugin:1.0-SNAPSHOT:advisorTool -DadviseVersion=jakarta10
```

Comment:
To continue my comments on the naming...

Another comment: the original kebab-case (`advisor-tool`) is more in-line with how other maven plugins are written, rather than what you're suggesting here (`advisorTool`).

I'd also suggest the goal name itself could also be changed: from `advisorTool` to simply `advise`

Reply:
thanks for comments, I already fixed